charts/dex.yaml

```diff
@@ -0,0 +1,102 @@
+image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
+
+nodeSelector:
+  node-role.kubernetes.io/master: ''
+
+tolerations:
+  - key: "node-role.kubernetes.io/bootstrap"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/master"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/infra"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+replicas: 2
+
+# grpc support
+grpc: false
+
+# https termination by dex itself
+https: false
+
+ports:
+  web:
+    containerPort: 5556
+    servicePort: 32000
+  # grpc:
+    # containerPort: 5000
+    # servicePort: 35000
+
+service:
+  type: ClusterIP
+
+ingress:
+  enabled: true
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: '/$2'
+    nginx.ingress.kubernetes.io/use-regex: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+    kubernetes.io/ingress.class: "nginx-control-plane"
+  path: /dex(/|$)(.*)
+  hosts:
+    - null
+  tls: []
+  #  - secretName: dex-example-tls <ROOT CERT AND ROOT KEY KUBERNETES SECRET>
+  #    hosts:
+  #      - dex.example.com
+
+certs:
+  web:
+    create: false
+  grpc:
+    create: false
+
+rbac:
+  create: true
+
+serviceAccount:
+  create: true
+  name:
+
+# ensure dex pods are running only on master nodes
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: "node-role.kubernetes.io/master"
+          operator: "Exists"
+
+config:
+  issuer: http://127.0.0.1:5556/dex
+  storage:
+    type: kubernetes
+    config:
+      inCluster: true
+  logger:
+    level: debug
+  web:
+    # port is taken from ports section above
+    address: 0.0.0.0
+    # tlsCert: /etc/dex/tls/https/server/tls.crt
+    # tlsKey: /etc/dex/tls/https/server/tls.key
+  connectors: {}
+
+  oauth2:
+    alwaysShowLoginScreen: false
+    skipApprovalScreen: true
+
+  expiry:
+    signingKeys: "6h"
+    idTokens: "24h"
+
+  enablePasswordDB: true
+  staticPasswords:
+    - email: "oidc_admin@metalk8s.com"
```

Comment:
There is no such domain. Need to find a better value for this, similar to our fake registry address.